graylog2-server/src/main/java/org/graylog2/shared/messageq/MessageQueueWriter.java

```diff
@@ -30,6 +31,9 @@
 
     void write(List<RawMessageEvent> entries) throws MessageQueueException;
 
+    default void preFlightCheck() throws PreflightCheckException {
+    }
+
```

Comment:
did you look at the cloud-plugin pr?

Reply:
Yep, I know that the Kafka message queue writer uses this. :slightly_smiling_face: I still think it would be nicer to decouple the pre-flight check logic from the `MessageQueueWriter` interface.

Reply:
Ok, done using a cast.